go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/eddiezane/captain-hook
 go 1.13
 
 require (
+	github.com/google/go-cmp v0.3.1
```

Comment:
This being used?

Reply:
From an earlier iteration. I guess go.mod didn't automatically clean up. Removed.